pylabrobot/resources/tecan/tip_racks.py

```diff
@@ -0,0 +1,1522 @@
+""" Tecan tip racks """
+
+# pylint: disable=empty-docstring
+# pylint: disable=invalid-name
+# pylint: disable=line-too-long
+
+from typing import List, Optional
+from pylabrobot.resources import TipRack, TipSpot, create_equally_spaced
+
+
+class TecanTipRack(TipRack):
+  """ Base class for Tecan tip racks. """
+
+  def __init__(
+    self,
+    name: str,
+    size_x: float,
+    size_y: float,
+    size_z: float,
+    z_travel: float,
+    z_start: float,
+    z_dispense: float,
+    z_max: float,
+    area: float,
+    items: Optional[List[List[TipSpot]]] = None,
+    category: str = "tecan_plate",
+    model: Optional[str] = None
+  ):
+    super().__init__(name, size_x, size_y, size_z, items, category=category, model=model)
+
+    self.z_travel = z_travel
+    self.z_start = z_start
+    self.z_dispense = z_dispense
+    self.z_max = z_max
+    self.area = area
+
+
+def DiTi_100ul_Te_MO(name: str) -> TecanTipRack:
+  return TecanTipRack(
+    name=name,
+    size_x=128.0,
+    size_y=85.80000000000001,
```

Comment:
These are most likely floating point errors, so maybe round to 4 decimal places when making resources?